src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -1222,15 +1264,27 @@ public void tryMarkRemoved(Topology topology, Id target)
         if (node.commandStores().count() == 0) return; // when starting up stores can be empty, so ignore
         Ranges ranges = topology.rangesForNode(target);
         if (ranges.isEmpty()) return;
-        tryMarkRemoved(ranges, 0).begin(node().agent());
+        long startNanos = Clock.Global.nanoTime();
+        exclusiveSyncPointWithRetries(ranges, 0)
```

Comment:
Is it OK for this to be async? Probably it is, and it always was, just making sure.

Reply:
its always been async, I just renamed it as I was supposed to add the durability message in, but since that was asked to remove I removed the `.flatMap` to send the message.

The logic has always been fire-and-forget and best-effort... durability scheduling is supposed to do the right thing but it takes time to do that, if you did another host replacement before it could do that we get into a bad state...

Reply:
also to directly answer, if we were blocking that wouldn't change anything.  There isn't an action we *should* take if this fails (other than log) and there is nothing that requires us to block for this...  its best effort to stay stable without depending on timing of durability scheduling.

Reply:
Sure, was just wondering really. Thank you for going into detail.